app/cafe.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+from typing import Any
+from app.errors import NotWearingMaskError, \
+    NotVaccinatedError, OutdatedVaccineError
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> Any:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitor is not vaccinated")
+        vaccine_expiration_date = visitor["vaccine"].get("expiration_date")
+        today = datetime.date.today()
+        if today > vaccine_expiration_date:
+            raise OutdatedVaccineError("Visitor's vaccine is outdated")
+        if not visitor["wearing_a_mask"]:

```

Comment:
use get() method instead of parenthesis everywhere for getting value by key to avoid KeyError